### PR TITLE
Unhandled messages

### PR DESCRIPTION
**Pull request by JJag**
Displays indicator on the message list if the message was not handled

### CODE REVIEW

File: monitoring/src/main/scala/akka/viz/aspects/ActorCellInstrumentation.scala [hunk 1]
```diff
@@ -21,7 +21,8 @@ class ActorCellInstrumentation {
   def message(jp: JoinPoint, msg: Any, me: ActorCell) {
     if (me.system.name != internalSystemName) {
       Thread.sleep(EventSystem.receiveDelay.toMillis)
-      EventSystem.report(Received(me.sender(), me.self, msg))
+      val canHandle = me.actor.receive.isDefinedAt(msg)
```
Comment (lustefaniak) [inline monitoring/src/main/scala/akka/viz/aspects/ActorCellInstrumentation.scala]: This won't work, as receive takes into account only static behaviour. Actor can become(behaviour) and I guess for that you would have to instrument Actor.unhandled, or subscribe to UnhandledMessage eventStream

**Comment by pkoryzna** [comment]
:heavy_check_mark:

File: frontend/src/main/scala/akka/viz/frontend/components/components.scala [hunk 2]
```diff
@@ -325,7 +332,7 @@ class MessagesPanel(selectedActors: Var[Set[String]]) extends Component with Pre
         table(
           cls := "table table-striped table-hover",
           thead(
-            tr(th("From"), th("To"), th("Class"))
+            tr(th(), th("From"), th("To"), th("Class"))
```
Comment (lustefaniak) [inline frontend/src/main/scala/akka/viz/frontend/components/components.scala]: Maybe put it somewhere inline, next to message class? That table is already too wide ;)

**Comment by lustefaniak** [comment]
:+1:

**Pull request merged**